reconcile/test/test_amtool.py

```diff
@@ -0,0 +1,27 @@
+from reconcile.utils import amtool
+from .fixtures import Fixtures
```

Comment:
why not absolute import path?

Reply:
A classic case on copy&pasta. Fixed in 3fc6aa98